### PR TITLE
Updated ChatGPT API + language argument support

### PR DESCRIPTION
**Pull request by laszlotuss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,27 +23,29 @@ pip3 install openai
...
+APP_CONTEXT = "This is my App, here is a summary what it does." # optional
 ```
 
 ### 2. Via Environment Arg
 
 Start the python script as follows
 
 ```bash
-OPENAI_API_KEY="put the token from OpenAI here" python3 filename.py
+OPENAI_API_KEY="put the token from OpenAI here" python3 translate_info.py
```
Comment (montioo) [inline README.md]: `translate_info.py` should not be executed, only the other files

File: translate_localization.py [hunk 2]
```diff
@@ -8,49 +8,77 @@
...
         return False
     
     def get_gpt_query(self) -> str:
-        query = f"key: {self.key}\n"
+        query = f"key: {self.key}\\n"
         comment = self.info_dict["comment"] if "comment" in self.info_dict else "No comment provided."
-        query += f"comment: {comment}\n"
-        query += "translation: \n"
+        query += f"comment: {comment}\\n"
+        query += "translation: \\n"
```
Comment (montioo) [inline translate_localization.py]: Why are you changing the newlines to escaped backslashes + n?

**Review by montioo — CHANGES_REQUESTED** [review]
Two minor remarks, other than that looks good. Thanks for contributing!

File: README.md (hunk 1 above)
Comment (laszlotuss) [reply README.md]: oh sorry, I'm not that familiar with Python and how environmental variables work, gonna restore it now

File: translate_localization.py (hunk 2 above)
Comment (laszlotuss) [reply translate_localization.py]: Probably I mass replaced something and I made this on mistake, restoring it now

**Comment by laszlotuss** [comment]
Thank you for your feedback, I restored both of my issued change as you requested.

Btw I also made more major modifications to better suit my need in my [development ](https://github.com/laszlotuss/l10n-gpt/tree/development) branch if you interested, but I'm still not experienced with Python practices and not sure about code quality. You are more than welcome to review or contribute if it suits your vision on this project.
_(I improved logging, had issues with max_attemps so made it an argument, and adjusted ChatGPT instructions which resulted better translations in my experience, especially for Hungarian :) )_

**Review by montioo — APPROVED** [review]
Looks good, thanks again!

**Comment by montioo** [comment]
Sounds good, I will have a look at your development branch in the beginning of the coming week :)

**Comment by montioo** [comment]
Hey @laszlotuss, I had a look at your development branch. The updated parameterization options and better logging look like good improvements. I will refine those updates a little with Python's `argparse` and `logging` modules and upload the changes in early May. I have some improvements to an iOS app that I want to localize and I will test the updated Python code before pushing.

**Comment by laszlotuss** [comment]
Thank you @montioo, I'm looking forward to it. 🫡

**Pull request merged**